docs/sanic/routing.md

```diff
@@ -41,24 +41,61 @@ inside the quotes. If the parameter does not match the specified type, Sanic
 will throw a `NotFound` exception, resulting in a `404: Page not found` error
 on the URL.
 
+### Supported types
+
+* string
+    * "Bob" or "Python 3" both would work here
```

Comment:
- I think you should take out the space and make it `Python3` which is more URL friendly.
- Remove the "would work here" and "all work here" language. I think it is unnecessary.
- How about putting the types in monospace? `string`, `int`, `number`...

Reply:
* I only mentioned the `Python 3` because during testing I realised that you *can* use spaces (and I believe more) in the string tag so it helped, in my opinion, to identify why you would use string over alpha.
* Hm fair enough. I only added in those to show what is valid in that system of type. Would you be fine with me leaving the examples in or would you suggest removing them or showing them differently? 
* OK

Reply:
Okay. You sold me on `Python 3` with the space 🍻

Reply:
Perhaps, I could adjust it so it becomes something similar to this
* `string`
    * "Bob"
    * "Python 3"
By just listing off examples of what works with the exception of alpha with including the note to say that symbols/non-alphanumeric would fail

Reply:
That works for me 💪

Reply:
Think this is now resolved with https://github.com/huge-success/sanic/pull/1620/commits/5ca1e4df4742217c26f1130a379ebb1a4844d64c :)